includes/admin/class-wc-rest-payments-terminal-locations-controller.php

```diff
@@ -45,17 +45,31 @@ public function register_routes() {
 	 * @return WP_REST_Response|WP_Error
 	 */
 	public function get_store_location( $request ) {
-		try {
-			$name    = get_bloginfo();
-			$address = [
-				'city'        => WC()->countries->get_base_city(),
-				'country'     => WC()->countries->get_base_country(),
-				'line1'       => WC()->countries->get_base_address(),
-				'line2'       => WC()->countries->get_base_address_2(),
-				'postal_code' => WC()->countries->get_base_postcode(),
-				'state'       => WC()->countries->get_base_state(),
-			];
+		$name             = get_bloginfo();
+		$store_address    = WC()->countries;
+		$location_address = [
```

Comment:
Not really this PR-related but may be worth changing. I noticed that if a merchant doesn't have something set in they's address, for instance, line2, then an error like that returned:

```
{
	"error": "parameter_invalid_empty",
	"message": "Error: You passed an empty string for 'address[line2]'. We assume empty values are an attempt to unset a parameter; however 'address[line2]' cannot be unset. You should remove 'address[line2]' from your request or supply a non-empty value."
}
```

It comes from Stripe, correct? Maybe it would make sense to not send fields that don't have data?

Reply:
Good catch @kidinov: fixed in 26bacd1a096856f3e55d91cddf5fa4f077a28165